index.js

```diff
@@ -106,3 +106,11 @@ btnBack.addEventListener('click', function () {
         console.log(nr2);
     }
 });
+
+const showHistory = document.querySelector('#show-history');
+const panelHistory = document.querySelector('.calc-history');
+
+showHistory.addEventListener("click", function () {
+    panelHistory.classList.toggle('hidden');
```

Comment:
Inside this function we should also change the text inside the Button.

Show History --> Hide History